### PR TITLE
temporary workaround for #156

### PR DESCRIPTION
**Pull request by rlamarche**
I called it a "temporary workaround" because I don't know sufficiently the project, but maybe it would work in all cases.

Fixes #165

### CODE REVIEW

**Comment by OskarStark** [comment]
Restarted travis

**Comment by OskarStark** [comment]
ping @rande

File: Entity/CategoryManager.php [hunk 1]
```diff
@@ -203,11 +203,7 @@ protected function loadCategories(ContextInterface $context)
         }
 
         foreach ($categories as $pos => $category) {
-            if ($pos === 0 && $category->getParent()) {
-                throw new \RuntimeException('The first category must be the root');
-            }
-
-            if ($pos == 0) {
+            if ($category->getParent() == null) {
```
Comment (Soullivaneuh) [inline Entity/CategoryManager.php]: * Yoda condition.
* Use strict comparison.

Comment (OskarStark) [inline Entity/CategoryManager.php]: @rlamarche can you please apply the comment of @Soullivaneuh

Comment (sagikazarmark) [inline Entity/CategoryManager.php]: Yoda condition? :smile: Haven't heard this expression earlier.

**Comment by core23** [comment]
ping @rlamarche

**Comment by rlamarche** [comment]
oops sorry, I'll document myself about yoda condition ;)

**Comment by core23** [comment]
Tahnk you @rlamarche

**Comment by core23** [comment]
Damn! Next time, please squash your commits please

**Pull request merged**